src/SchemaRegistry.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
I'm opening an issue #23 to decide which licence to use.
Not to be fixed in this PR to avoid additional lead time validation